lib/widget/input/dropdown.dart

```diff
@@ -179,14 +185,28 @@ class SelectOneController extends FormFieldController {
   dynamic maybeValue;
   int gap = 0;
   bool autoComplete = false;
+  final StreamController<dynamic> fieldController = StreamController<dynamic>();
 
   framework.EnsembleAction? onChange;
 }
 
 class SelectOneState extends FormFieldWidgetState<SelectOne> {
   final focusNode = FocusNode();
+  late TextEditingController rawTextEditingController;
+  late StreamSubscription<dynamic> streamSubscription;
+
   @override
   void initState() {
+    rawTextEditingController = TextEditingController();
+    streamSubscription =
```

Comment:
This pattern is OK but in Controller we already have a way to invoke the widgetState's setState(). Just use `notifyListeners()`.